ui/app/controllers/variables/variable/index.js

```diff
@@ -6,6 +6,12 @@ import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
 export default class VariablesVariableIndexController extends Controller {
+  queryParams = [
+    {
+      view: 'view',
+    },
+  ];
```

Comment:
`nit`: This doesn't need an alias. When we have a query parameter `namespace` we need to alias that because `namespace` is a reserved keyword in Ember.

you good with:

```
queryParams = ['view'];
@tracked view = 'table';
```